src/status_im/chat/subs.cljs

```diff
@@ -213,3 +218,9 @@
          (filter :outgoing)
          (sort-by :clock-value >)
          (first))))
+
+(reg-sub :get-message-preview
+   (fn [db [_ message-id]]
+     (when-let [preview (get-in db [:message-data :preview message-id :markup])]
+       (commands-utils/generate-hiccup preview))))
```

Comment:
it will be generate-hiccup all the time db is changing, better to create a separate subscription with extracting markup and use it in this one